code/modules/spells/spell_types/godhand.dm

```diff
@@ -186,8 +187,4 @@
 		*/
 		M.forceMove(pie)
 
-
-		charges--
-
-	if(charges <= 0)
-		qdel(src)
+	. = ..()
```

Comment:
You need to clean `/obj/item/melee/touch_attack/bspie/afterattack`. It still has playsound and the parent proc call position doesn't seems to be okay.

Reply:
will move parent inside the do_mob